pkg/metadata/metadata.go

```diff
@@ -79,8 +79,7 @@ func (a *Service) ChartName() string {
 func (a *Service) TemplatedName(name string) string {
 	_, contains := a.names[name]
 	if !contains {
-		// template only app objects
-		return name
```

Comment:
Please leave it as it is. If an object name is not part of the current chart it should not be templated

Reply:
Added a new method instead of using the existing one.

Reply:
Added a new method instead of using this one.